code-of-conduct.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+https://docs.silverstripe.org/en/3.1/contributing/code_of_conduct/
```

Comment:
Make this https://docs.silverstripe.org/en/contributing/code_of_conduct/ so it's version agnostic :)